### PR TITLE
TL Localization - Chapter 6, section 2 - part 2 (1,499 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/2-contributing.asc [hunk 1]
```diff
@@ -150,115 +150,115 @@ Kapag ang tagapangasiwa ay gumagawa ng komentong ito, ang taong nagbukas ng Kahi
...
 
-[NOTE]
-.Not Only Forks
+[TANDAAN]
+.Hindi Lamang Forks
 ====
-It's important to note that you can also open a Pull Request between two branches in the same repository. If you're working on a feature with someone and you both have write access to the project, you can push a topic branch to the repository and open a Pull Request on it to the `master` branch of that same project to initiate the code review and discussion process. No forking necessary.
+Mahalaga na tandaan na maaari ka ring magbukas ng isang Kahilingan na Pull sa pagitan ng dalawang branch sa parehong repositoryo. Kung ikaw ay nagtatrabaho sa isang tampok at pareho kayong may access sa pagsulat sa proyekto, maaari kang mag-push ng branch ng paksa repositoryo at magbukas ng isang Kahilingan na Pull Request dito sa branch na `master` sa parehong proyekto na iyon upang masimulan ang pagsusuri sa review at proseso ng talakayin. Walang kinakailangang pag-fork.
```
Comment (josteinfrancois) [inline book/06-github/sections/2-contributing.asc]: remove the word `Request` in `Kahilingan na Pull Request`

File: book/06-github/sections/2-contributing.asc [hunk 2]
```diff
@@ -150,115 +150,115 @@ Kapag ang tagapangasiwa ay gumagawa ng komentong ito, ang taong nagbukas ng Kahi
...
 ====
 
-==== Advanced Pull Requests
+==== Mga Advanced na Kahilingan na Pull
 
-Now that we've covered the basics of contributing to a project on GitHub, let's cover a few interesting tips and tricks about Pull Requests so you can be more effective in using them.
+Ngayon na ating natalakay ang mga pangunahing kaalaman sa pag-aambag sa isang proyekto sa GitHub, talakayin natin ang ilang mga kagiliw-giliw na tip at kaalaman tungkol sa mga Kahilingan na Pull upang ikaw ay mas magiging epektibo sa paggamit nito.
 
-===== Pull Requests as Patches
+===== Mga Kahilangan na Pull bilang Patches
```
Comment (josteinfrancois) [inline book/06-github/sections/2-contributing.asc]: please change `Patches` to `mga Patch`

File: book/06-github/sections/2-contributing.asc [hunk 3]
```diff
@@ -150,115 +150,115 @@ Kapag ang tagapangasiwa ay gumagawa ng komentong ito, ang taong nagbukas ng Kahi
...
 
-It's important to understand that many projects don't really think of Pull Requests as queues of perfect patches that should apply cleanly in order, as most mailing list-based projects think of patch series contributions. Most GitHub projects think about Pull Request branches as iterative conversations around a proposed change, culminating in a unified diff that is applied by merging.
+Mahalagang maintindihan na maraming mga proyekto ang hindi talaga nag-iisip ng Kahilangan na Pull bilang mga queue ng perpektong mga patch na dapat i-apply nang malinis sa pagkakasunud-sunod, tulad ng karamihan sa mga proyekto nakabatay sa listahan ng mail na iniisip ang mga kontribusyon ng serye ng patch. Iniisip ng karamihan sa mga proyekto ng GitHub ang mga branch ng Kahilingan na Pull bilang umuulit na pag-uusap sa paligid ng isang ipinanukalang pagbabago, nagtatapos sa isang pinag-isang diff na inilalapat sa pamamagitan ng pagsasama.
```
Comment (josteinfrancois) [inline book/06-github/sections/2-contributing.asc]: could change `i-apply` to `malapat`

File: book/06-github/sections/2-contributing.asc [hunk 4]
```diff
@@ -150,115 +150,115 @@ Kapag ang tagapangasiwa ay gumagawa ng komentong ito, ang taong nagbukas ng Kahi
...
 
-You have two main options in order to do this. You can either rebase your branch on top of whatever the target branch is (normally the `master` branch of the repository you forked), or you can merge the target branch into your branch.
+Mayroon kang dalawang pangunahing mga opsyon upang magawa ito. Maaari mong i-rebase ang iyong branch sa itaas ng anumang ang branch na target (karaniwan ang branch na `master` repositoryo na iyong na-fork), o maaari mong isama ang branch na target sa iyong branch.
 
-Most developers on GitHub will choose to do the latter, for the same reasons we just went over in the previous section. What matters is the history and the final merge, so rebasing isn't getting you much other than a slightly cleaner history and in return is *far* more difficult and error prone.
+Pipiliin ng karamihan sa mga developer sa GitHub na gawin ang paghuli, para sa parehong dahilan na ating tinalakay sa nakaraang seksyon. Ang mahalaga ay ang kasaysayan at ang panghiling merge, kay ang pag-rebase ay hindi nagkukuha sa iyo ng marami maliban sa isang bahagyang mas malinis na kasaysayan at sa pagbabalik ay *malayo* na mas mahirap at madaling kapitan ng error.
```
Comment (josteinfrancois) [inline book/06-github/sections/2-contributing.asc]: - change `panghiling` to `panghuling`
- `kay` to `kaya`

File: book/06-github/sections/2-contributing.asc [hunk 5]
```diff
@@ -150,115 +150,115 @@ Kapag ang tagapangasiwa ay gumagawa ng komentong ito, ang taong nagbukas ng Kahi
...
 
-If you want to merge in the target branch to make your Pull Request mergeable, you would add the original repository as a new remote, fetch from it, merge the main branch of that repository into your topic branch, fix any issues and finally push it back up to the same branch you opened the Pull Request on.
+Kung gusto mong mag-merge sa branch ng target upang magawa mong maaari i-merge ang iyong Kahilingan na Pull,idadagdag mo ang orihinal na repositoryo bilang isang bagong remote, mag-fetch mula dito, mag-merge the pangunahing branch ng repositoryo na iyon sa iyong branch ng paksa, mag-ayos ng anumang mga isyo at panghuli ay i-push ito sa parehong branch kung saan mo binuksan ang Kahilingan na Pull.
```
Comment (josteinfrancois) [inline book/06-github/sections/2-contributing.asc]: - add space after the comma in `Pull,idadagdag`
- `mag-merge the pangunahing` to `mag-merge ng pangunahing`

**Review by josteinfrancois — COMMENTED** [review]
Hello @heywens. I have reviewed your work. Please see the comments I placed.

**Comment by heywens** [comment]
hi @josteinfrancois, thanks for the review. necessary changes have been applied.

**Pull request merged**